src/lib/types.ts

```diff
@@ -382,6 +425,11 @@ export interface ApiTLSOptions {
   versions?: string[]
 }
 
+export enum MainStatusCode {
+  Unknown = 'unknown',
+  CapacityErr = 'insufficient_capacity',
```

Comment:
```suggestion
  InsufficientCapacity = 'insufficient_capacity',
```

nitpick, in case fly adds other types of capacity errors